lib/alertStateMachine.js

```diff
@@ -15,7 +16,11 @@ class AlertStateMachine {
     switch (currentAlertState) {
       case CHATBOT_STATE.STARTED:
       case CHATBOT_STATE.WAITING_FOR_REPLY:
-        nextAlertState = CHATBOT_STATE.WAITING_FOR_CATEGORY
+        if (messageText.trim().toLowerCase() === this.getClientMessageForRequestToReset(language)) {
```

Comment:
Nope; I'll test translations in test cases :)

Reply:
I've modified the dummy functions and respective test cases to use the language argument passed, so that covers translations on the brave-alert-lib side of things. It's up to BraveSensor or BraveButtons to make this argument useful :)